### PR TITLE
Added pause-resume functionality in the install.sh file

### PR DESCRIPTION
**Pull request by dope-dependent**
I have added pause-resume functionality in the `install.sh` file. The following changes have been done.

1. **Pause resume functionality** : As the user enters their details for the database, they are simultaneously stored in a `.variable.txt` hidden file. If the user presses `Ctrl+C`anytime (i.e. aborts the program), and starts from the beginning (running the `install.sh` file again), they are asked whether they want to start afresh or use that backup and the program proceeds. After all the inputs are taken, the backup file is removed.
2. **Slight UI modification** has also been done to make the process look smoother.
3. **About other sub-issues (within #34)** : @pasthorizon is writing code to merge the `install.sh` and the `init.sh` into one single file (`install.sh`)

### CODE REVIEW

**Review by grapheo12 — APPROVED** [review]
Love this work!!!

**Comment by grapheo12** [comment]
@dope-dependent Please rebase your work to the current master branch.

**Comment by dope-dependent** [comment]
Hey @grapheo12, thanks for approving. I am completely new to open source and only know a little git. Can you tell my what "rebasing" means in layman terms?

**Comment by dope-dependent** [comment]
Does it mean that I have to update my install.sh file (due to the main repo being updated)?

**Comment by grapheo12** [comment]
@dope-dependent Read below carefully!


See the `master` branch has moved ahead after you created your `pause-resume` branch.
Particularly, the `install.sh` file has changed and git doesn't know whose changes to keep, yours or the current one.
What you need to do is:

```bash
git checkout master                                     # Go to the master branch
git pull upstream master                                # I am assuming that you've named this repo upstream and your fork as origin
git checkout pause-resume                               # Go to pause-resume again
git merge master                                        # Try to merge master branch to your branch, this will fail partially
```

GIt will tell you that you got merge conflicts. Open the project (preferably) in VS Code. Go to the `install.sh` file.
You'll see that areas with Merge conflicts will be merged. Use your judgment and select the changes you want to keep.
You can keep both (safest) then rectify the code as per your needs.

Then commit and push to your branch.
Also, go back to your local master branch and push it to `origin master`.

This will complete the merge conflict resolution. This isn't rebasing so to speak. But the effect is same.

**Comment by dope-dependent** [comment]
Thanks @grapheo12, I will do the necessary changes by tomorrow morning ETA.

**Comment by dope-dependent** [comment]
Hey @grapheo12, is it fine now? I have done as you had told.
